src/main/java/io/cryostat/net/web/http/api/beta/RecordingGetWithJwtHandler.java

```diff
@@ -107,6 +119,23 @@ public boolean isAsync() {
         return true;
     }
 
+    @Override
+    public SecurityContext securityContext(RequestParameters params) {
+        String sourceTarget = params.getPathParams().get("sourceTarget");
+        String recordingName = params.getPathParams().get("recordingName");
+        try {
+            String id = jvmId.getJvmId(sourceTarget);
+            String encoded = base32.encodeToString(id.getBytes(StandardCharsets.UTF_8));
```

Comment:
This can use `String encoded = jvmId.jvmIdToSubdirectoryName` without the extra injection of base32.